src/components/playback/playbackmanager.js

```diff
@@ -1868,48 +1868,37 @@ class PlaybackManager {
                 }, queryOptions));
             } else if (firstItem.Type === 'Series' || firstItem.Type === 'Season') {
                 const apiClient = ServerConnections.getApiClient(firstItem.ServerId);
+                const isSeason = firstItem.Type === 'Season';
 
                 promise = apiClient.getEpisodes(firstItem.SeriesId || firstItem.Id, {
                     IsVirtualUnaired: false,
                     IsMissing: false,
+                    SeasonId: isSeason ? firstItem.Id : undefined,
```

Comment:
Iirc, the idea was to continue playing the series (by starting a single season), including the next seasons.
Currently, this limits queue to the one season.

Reply:
I'd be in favor of adding an explicit option to start playing a series at a particular season. Something like a `Play From Here` in the drop down.
![image](https://github.com/jellyfin/jellyfin-web/assets/6550543/377049d5-d0f3-47cc-9618-4069824d0695)

I do not think it should be the default however, maybe a library-wide option instead? Automatically starting the next season doesn't always make sense when, for example, someone clicks play on a `Specials` season. The expectation as of 10.8.13 is that only the `Specials` season would play. 
![image](https://github.com/jellyfin/jellyfin-web/assets/6550543/2105779b-7641-43e1-8e5e-eaf44bab8e84)

Also doesn't make sense for shows with self-contained seasons.
![image](https://github.com/jellyfin/jellyfin-web/assets/6550543/0837e7e3-825d-421f-a320-0e737069ade5)

Reply:
Related PR #4827

Reply:
I didn't realize this, but there's already an option to `Play all from here` shown on individual episodes in individual seasons. That option could likely be added to show on the season's drop down as well - thus implementing the behavior you're talking about.